### PR TITLE
adding about us page

### PR DESCRIPTION
**Pull request by SanjanaSogimatt**
## 📋 Description

Adding about us page

## Release Notes

- **New Features**
  - Updated URLs for the "Partnership" and "Products" menu items to direct users to the new blog section.
  - Revamped "About" section with new content, metrics, and team member profiles, enhancing user engagement.

- **Bug Fixes**
  - Removed unnecessary button elements from various components for a cleaner user interface.

- **Style**
  - Improved styling across components for better visual presentation and consistency, including new gradient backgrounds and responsive layouts.

- **Chores**
  - Removed outdated images and components to streamline the application.

### CODE REVIEW

File: public/images/Team/Team_DPH.xlsx
Comment (drtechie) [inline public/images/Team/Team_DPH.xlsx:1]: let's remove this file.

File: src/config/menu.json [hunk 1]
```diff
@@ -52,7 +52,7 @@
     },
     {
       "name": "Partnership",
-      "url": "/contact"
+      "url": "/blog"
```
Comment (drtechie) [inline src/config/menu.json:55]: Partnership will go to /blog? 🤔

Comment (SanjanaSogimatt) [reply src/config/menu.json:55]: It's just for reference I'll change it later

**Review by drtechie — APPROVED** [review]
Approving and merging this for now.

Will have to do the following in a follow-up PR.

1. the team images are huge (2MB+!). We should make it smaller and optimize with webp images.
2. Team member images have different aspect ratios which will have to be standardized.

**Pull request merged**